docs/concepts/00_what_is_aries.md

```diff
@@ -0,0 +1,36 @@
+# What is Aries?
```

Comment:
honestly, I find this doc rather confusing because I get the impression people will easily mistake its contents for a description of the capabilities of *aries-framework-go* instead of the *Hyperledger Aries* project.

Reply:
true true, i'll change it to Hyperledger Aries